<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QPDFWalker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/app_icon</normaloff>:/icons/app_icon</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="uiFilesList"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="uiOpenFile">
         <property name="text">
          <string>&amp;Open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="uiCloseFile">
         <property name="text">
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="uiWalk">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Walk</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="uiQuit">
         <property name="text">
          <string>&amp;Quit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="uiAbout">
         <property name="text">
          <string>&amp;About</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
